src/ofxKinectForWindows2/Source/Depth.h

```diff
@@ -31,8 +33,12 @@ namespace ofxKinectForWindows2 {
 
 			ofFloatPixels getColorToWorldMap(int colorImageWidth = 1920, int colorImageHeight = 1080) const;
 			ofFloatPixels getDepthToWorldMap() const;
+			UINT16				getMinDepthReliableDistance() const;

```

Comment:
maybe add a comment, what units are these in 
e.g.:
`UINT16 getMinDepthReliableDistance() const; /// get minimum depth range in mm`

Also you've put a TAB between the UINT16 and teh getMinDepthReliableDistance

Also i'd use `unsigned short` rather than `UINT16`. Although these are exactly the same, `unsigned short` matches the style elsewhere.

Thank you!